src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -496,19 +496,21 @@ public void testPanelsCollapseExpand() {
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(true));
+        assertTrue(resultsPage.areResultsVisible());
 
         resultsPage.clickCollapseExpand();
+        resultsPage.waitForPanelsToCollapse();
         assertEquals("Expand Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Expand all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(false));
+        assertTrue(resultsPage.areResultsHidden());
 
         resultsPage.clickCollapseExpand();
+        resultsPage.waitForPanelsToExpand();
         assertEquals("Collapse Questions", resultsPage.collapseExpandButton.getText());
         assertEquals("Collapse all panels. You can also click on the panel heading to toggle each one individually.",
                      resultsPage.collapseExpandButton.getAttribute("data-original-title"));
-        assertTrue(resultsPage.verifyAllResultsPanelBodyVisibility(true));
+        assertTrue(resultsPage.areResultsVisible());
```

Comment:
What I meant was:
```java
/**
 * Boolean-returning method, named is* or are* or ...
 */
public boolean isCondition() {
	return condition;
}

/**
 * Asserting method, named verify*
 */
public void verifyCondition() {
	assertTrue(condition);
}
```
I suggested 2 instead because all the usages of the boolean method involve `assertTrue(...)` only. If there are some `assertFalse(...)` usages then it's right that it has to be 1. You could consider doing the same for the other PR.